docs/OV_Runtime_UG/integrate_with_your_application.md

```diff
@@ -11,71 +11,69 @@
    openvino_docs_OV_UG_Python_API_exclusives
    openvino_docs_MO_DG_TensorFlow_Frontend
 
-@endsphinxdirective
 
+Following these steps, you can implement a typical OpenVINO™ Runtime inference 
+pipeline in your application. Before proceeding, make sure you have 
+:doc:`installed OpenVINO Runtime <openvino_docs_install_guides_install_runtime>` and set environment variables (run ``<INSTALL_DIR>/setupvars.sh`` for Linux or ``setupvars.bat`` for Windows, otherwise, the ``OpenVINO_DIR`` variable won't be configured properly to pass ``find_package`` calls).
 
-Following these steps, you can implement a typical OpenVINO™ Runtime inference pipeline in your application. Before proceeding, make sure you have [installed OpenVINO Runtime](../install_guides/installing-openvino-runtime.md) and set environment variables (run `<INSTALL_DIR>/setupvars.sh` for Linux or `setupvars.bat` for Windows, otherwise, the `OpenVINO_DIR` variable won't be configured properly to pass `find_package` calls).
 
+.. image:: _static/images/IMPLEMENT_PIPELINE_with_API_C.svg
 
-![ie_api_use_cpp]
 
-## Step 1. Create OpenVINO™ Runtime Core 
+Step 1. Create OpenVINO Runtime Core
+####################################
 
 Include next files to work with OpenVINO™ Runtime:
 
-@sphinxtabset
 
-@sphinxtab{C++}
-
-@snippet docs/snippets/src/main.cpp include
-
-@endsphinxtab
-
-@sphinxtab{Python}
+.. tab:: C++
 
-@snippet docs/snippets/src/main.py import
+   .. doxygensnippet:: docs/snippets/src/main.cpp
```

Comment:
the tabs are not synchronized. Please, checks docs once they are compiled (download archive from there https://github.com/openvinotoolkit/openvino/actions/runs/4324444656)
<img width="687" alt="Screenshot 2023-03-03 at 7 40 32 PM" src="https://user-images.githubusercontent.com/2566854/222762879-4f5268e0-65f8-4f57-b1b5-d97823fd55ef.png">

Reply:
Have fixed it: https://github.com/openvinotoolkit/openvino/actions/runs/4340623643